### ISSUE TITLE
Build folder should be ignored by git

### ISSUE THREAD

**Issue by imantahriri** [origo-map/origo#596]
build directory should be added to .gitignore file. Otherwise with every build more than 100 files comes up in the git bash as "modified" which is quite irritating.

**Comment by tonnyandersson** [comment]
@imantahriri If we just add the build directory to the .gitignore file, we will have to remove it every time we make a new release in order to include the build files in the release, and then put it back in afterwards. This may not seem like a big deal since we're not making a lot of new releases, but it is not a particularly elegant solution.

Instead, we are thinking about introducing a new build directory - _build-dev_, or something - which is created when running the standard build script. This directory should be added to the .gitignore file. In addition, we will create a new build script, specifically for making release builds. Release build files will end up in the old build directory (possibly renamed to _build-release_ or similar), which isn't included in the .gitignore file.

So, to summarize:

- Your standard everyday build files will end up in a build-dev directory and will be ignored by git
- Our release build files will end up in the build (or build-release) directory and will _not_ be ignored by git

How does that sound?

**Comment by imantahriri** [comment]
Sounds reasonable.
